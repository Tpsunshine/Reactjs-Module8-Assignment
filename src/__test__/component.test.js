import React from 'react';
import {create} from 'react-test-renderer';
import ListView from '../container/listView';

describe('Listview Component Snapshot', ()=>
{
    test("ListView",()=>
    {
        let tree = create[<ListView/>];
        expect(tree).toMatchSnapshot();
    })
})