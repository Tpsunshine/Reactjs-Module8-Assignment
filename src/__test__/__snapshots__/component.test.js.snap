// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Listview Component Snapshot ListView 1`] = `undefined`;
